app/design/frontend/Gumennoy/FirstTheme/web/css/source/_typography.less

```diff
@@ -0,0 +1,6 @@
+.lib-font-face(
```

Comment:
The file `/app/design/frontend/Gumennoy/FirstTheme/web/css/source/_typography.less`can not be used to add new font because there is such file in the Magento/blank theme - https://github.com/magento/magento2/blob/2.2.6/app/design/frontend/Magento/blank/web/css/source/_typography.less
This means that the opensans font is now not loaded! You should use one of these two ways to load this new font:
1) keep the content of the original file
2) add your font to the `_extend.less` file